bssl_wrapper/bssl_wrapper.h

```diff
@@ -45,11 +45,11 @@ class StackAllocated {
 
   void Reset() {
     cleanup(ctx_);
-    init(ctx_);
+    init();
```

Comment:
don't you need to do `ctx_ = init()` here?

Reply:
No, pls. see https://github.com/Maistra/bssl_wrapper/pull/3/files#diff-1c12f051fb284995269056bf27f1382aR30

Reply:
Hmm, let me have another look...

Reply:
Yep, you are right. Fixed, good catch.